com.regnosys.rosetta.tests/src/com/regnosys/rosetta/generator/java/docrefs/DocReferenceTest.xtend

```diff
@@ -127,7 +127,7 @@ class DocReferenceTest {
 			func Sum:
 				[docReference Org1 Agr1 name "something"]
 				inputs: x number (0..*)
-				output: sum number (1..1)
+				output: s number (1..1)
```

Comment:
`sum` is a keyword and should not be used as a variable name. Otherwise there is an ambiguity in the parser. E.g., should `sum` be interpreted as the variable `sum` or as the operation `sum` applied to an implicit variable?

Alternative is to disallow applying `sum` to an implicit variable (e.g., you always need to specify the input: `it sum`)